websites/Z/Zee5/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "993170607841746974",
+		"name": "பில்லி கசாப்புக்காரன்"
+	},
+	"service": "Zee5",
```

Comment:
It looks like the official spelling is in all caps, maybe change the service and folder name to match it.

Reply:
And ofc the discord app name.